java/code/src/com/suse/manager/webui/controllers/utils/test/SSHMinionBootstrapperTest.java

```diff
@@ -82,6 +82,7 @@ protected String getDefaultContactMethod() {
     protected Map<String, Object> createPillarData(Optional<ActivationKey> key) {
         Map<String, Object> pillarData = new HashMap<>();
         pillarData.put("mgr_server", ConfigDefaults.get().getCobblerHost());
+        pillarData.put("root_server", ConfigDefaults.get().getCobblerHost());
```

Comment:
Is the additional pillar data value really needed in those two cases where we set it to the exactly same value as `mgr_server`?

Reply:
This seems to be a test. I do not think we should omit one if both are the same.